tests/integrate_test/cita_charge_mode.sh

```diff
@@ -1,113 +1,100 @@
 #!/bin/bash
 set -e
 
-if [[ `uname` == 'Darwin' ]]
+if [[ $(uname) == 'Darwin' ]]
 then
-    SOURCE_DIR=$(realpath $(dirname $0)/../..)
+    SOURCE_DIR=$(realpath "$(dirname "$0")"/../..)
 else
-    SOURCE_DIR=$(readlink -f $(dirname $0)/../..)
+    SOURCE_DIR=$(readlink -f "$(dirname "$0")"/../..)
 fi
 BINARY_DIR=${SOURCE_DIR}/target/install
 
-################################################################################
-echo -n "0) prepare  ...  "
-. ${SOURCE_DIR}/tests/integrate_test/util.sh
-cd ${BINARY_DIR}
-echo "DONE"
+test_charge_mode() {
+    local version=$1
+    local node_0_privkey
+    local node_1_privkey
+    local node_2_privkey
+    local node_3_privkey
+    node_0_privkey=$(cat ./"$CHAIN_NAME"/0/privkey)
+    node_1_privkey=$(cat ./"$CHAIN_NAME"/1/privkey)
+    node_2_privkey=$(cat ./"$CHAIN_NAME"/2/privkey)
+    node_3_privkey=$(cat ./"$CHAIN_NAME"/3/privkey)
+    cd ./scripts/txtool/txtool
```

Comment:
local node_0_privkey = $(cat ./"$CHAIN_NAME"/0/privkey) ?

Reply:
this will cause warning of shellcheck.